release-notes/4.7.20.md

```diff
@@ -0,0 +1,252 @@
+# CiviCRM 4.7.20
+
+Released June 7, 2017
+
+- **[Features](#features)**
+- **[Bugs resolved](#bugs)**
+- **[Miscellany](#misc)**
+- **[Credits](#credits)**
+
+## <a name="features"></a>Features
+
+### Core CiviCRM
+
+- **[CRM-20576](https://issues.civicrm.org/jira/browse/CRM-20576) Increase length of civicrm_batch.title ([10359](https://github.com/civicrm/civicrm-core/pull/10359))**
+
+- **[CRM-20238](https://issues.civicrm.org/jira/browse/CRM-20238) Hook for inbound SMS messages ([10353](https://github.com/civicrm/civicrm-core/pull/10353))**
```

Comment:
@seamuslee001 It got scooped up because https://github.com/civicrm/civicrm-core/pull/10353 is in 4.7.20-rc.  I'll mark that item as "preliminary work".